satpy/FY4A_agri_introduction(EN).ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>In the comment on the first line of this cell, it would be nice to have a link to the satpy FAQ where we explain how to tune memory/cpu usage ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Added.</p>